src/package/model-config.ts

```
@@ -1,12 +1,20 @@
+<<<<<<< HEAD:src/model-config.ts
 export type ModelName = "Orpheus-3b" | "gpt-4o-realtime-preview-2024-12-17" | "gpt-4o-mini-realtime-preview-2024-12-17";
+import { FunctionDefinition } from "./tools";
 
+type FunctionDefinition = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FunctionDefinition'. Consider to delete it or rename it.

'FunctionDefinition' is defined here:


(lint/suspicious/noRedeclare)